guides/common/modules/con_understanding-package-mode-and-image-mode-hosts.adoc

```diff
@@ -0,0 +1,31 @@
+[id="understanding_package_mode_and_image_mode_hosts_{context}"]
+= Understanding Package mode and Image mode hosts
+
+In the latest update, hosts can now operate in two modes: package mode and image mode.
+
+.Image mode hosts
+Image mode hosts include eight new fields that reflect their boot status.
+These fields track the container images used in various scenarios:
+
+* *Booted Image & Digest*: Indicates the container image the host is currently using.
+* *Staged Image & Digest*: Indicates the container image the host uses after the next reboot.
+* *Available Image & Digest*: Represents the container image cached by running `Bootc upgrade --check`.
+* *Rollback Image & Digest*: Represents the container image the host reverts to after the next reboot if `Bootc rollback` is applied.
+
+.All host listing page updates
+* *New Column for Host Types*: The host listing page includes a column with icons that indicate the host type.
```

Comment:
Note: this is only on the _new_ hosts overview page, which is not the default yet.